src/engine/SCons/Tool/install.py

```diff
@@ -267,21 +282,49 @@ def Dir(self, name):
         name = SCons.Util.make_path_relative(name)
         return self.dir.Dir(name)
 
+
+def install_command_generator(env, target, source, for_signature=False):
+    if env.get('INSTALL', False):
+        return "$INSTALLFUNC"
+
+    if not SCons.Util.is_List(target):
+        target = [target]
+
+    if not SCons.Util.is_List(source):
+        source = [source]
+
+    if not for_signature:
+        for t in target:
+            if t.isfile():
+                t.remove()
+
+    if len(target) == 1:
+        copyingdirs = [s for s in source if s.isdir()]
+        if copyingdirs:
+            return '$INSTALLDIRCOPY $INSTALLDIRCOPYFLAGS $SOURCES/ $TARGET'
```

Comment:
I'm not sure if the `/` is necessary? It seems like it could lead to some weird situations when there are more than one source.

```python
env.Install(
    target="public",
    source=[
          "web/html",
          "web/js",
          "web/css",
    ]
)
```

Would expand to something like `cp --recursive web/js web/html web/css/ public`? Which would still work but it would also work without this weird extra slash.

It's possible depending on how you want this to work you may need to specify `$TARGET/` however.

Reply:
The current /'s allow install and installAs to work as they did previously.
And with your example above it works as expected:
```
scons: Reading SConscript files ...
scons: done reading SConscript files.
scons: Building targets ...
cp --preserve=all --recursive web/css/. public/css
cp --preserve=all --recursive web/html/. public/html
cp --preserve=all --recursive web/js/. public/js
scons: done building targets.
```

```
$ tree web/
web/
├── css
│   ├── file1
│   ├── file2
│   └── subdir
│       └── file3
├── html
│   ├── file1
│   ├── file2
│   └── subdir
│       └── file3
└── js
    ├── file1
    ├── file2
    └── subdir
        └── file3
```

```
$ tree public/
public/
├── css
│   ├── file1
│   ├── file2
│   └── subdir
│       └── file3
├── html
│   ├── file1
│   ├── file2
│   └── subdir
│       └── file3
└── js
    ├── file1
    ├── file2
    └── subdir
        └── file3
```

Reply:
LGTM